src/components/backgroundselector/partials/backgroundselector.html

```diff
@@ -4,6 +4,10 @@
      translate-attr="{title: layer.label}">
   <div class="ga-bg-layer-text" translate>{{layer.label}}</div>
   <button class="ga-icon ga-btn fa fa-user" ng-click="showWarning(layer)" ng-show="!!layer.styleUrl"></button>
+  <div class="ga-open-edit-bt" translate-attr="{title: 'open_edit_bt_title'}"
+       ng-click="activeEdit(layer)">
+    <div class="fa fa-ga-mapfunction"></div>
+  </div>
```

Comment:
`ng-click="activeEdit(layer)"` should probably be a `toggleEdit`

Reply:
Yes why not